.rubocop.yml

```diff
@@ -10,5 +10,8 @@ Metrics/PerceivedComplexity:
 Metrics/MethodLength:
   Max: 20
 
+Metrics/LineLength:
+  Max: 150
```

Comment:
I would hesitate to add this. Many open source projects follow the 80 character max width. I fear that by adding this, the value will continue to grow overtime and thus be useless.

The question to ask is, can the line in question be shortened without drastically decreasing readability?